tests/caasp/stack_conformance.pm

```diff
@@ -21,29 +21,48 @@ use testapi;
 use caasp 'script_retry';
 use version_utils 'is_caasp';
 
+my $config_json = <<'EOF';
+{
+  "plugins": [ { "name": "e2e" } ]
+}
+EOF
+
 sub run {
-    switch_to 'xterm';
+    # The repository for the sonobuoy package
+    my $repo = "https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Tumbleweed/devel:kubic.repo";
 
-    # CaaSP 2.0 has Kubernetes 1.8.9 which doesn't work with v1.10.0 testsuite
-    my $branch = is_caasp('qam') ? '6179d790e6bfc799afef5058ce50a2f314983fa2' : 'master';
+    my $json_name = "sonobuoy.json";
+    my $logs_dir  = "sonobuoy_logs";
 
-    # https://github.com/cncf/k8s-conformance/blob/master/instructions.md
-    my $sb_yaml = "https://raw.githubusercontent.com/cncf/k8s-conformance/$branch/sonobuoy-conformance.yaml";
-    my $sb_exit = '"no-exit was specified, sonobuoy is now blocking"';
     my $sb_pass = '"SUCCESS! -- [1-9][0-9]\+ Passed | 0 Failed | 0 Pending.*PASS"';
     my $sb_test = '"Test Suite Passed"';
 
-    # Run conformance tests and wait 90 minutes for result
-    assert_script_run "curl -L $sb_yaml | kubectl apply -f -";
-    script_retry "kubectl -n sonobuoy logs sonobuoy | grep $sb_exit", retry => 90, delay => 60;
+    switch_to 'xterm';
+    become_root;
+
+    # Add the repo
+    zypper_call " ar -Gf $repo";
+
+    # Install the sonobuoy package on the controller node
+    zypper_call "-n in sonobuoy";
+
```

Comment:
add disable/ remove sonobuy repo will be nice

Reply:
this conformance test is always the last one that runs, and the cluster is destroyed afterwards, so there will be no issues